main.go

```diff
@@ -10,8 +10,10 @@ import (
 )
 
 var (
-	pythonPath = os.Getenv("HOME") + "/.local/share/gosplat/src/python_helper/fast_model_compare.py"
-	modelBin   = os.Getenv("HOME") + "/.local/share/gosplat/fast-fb-model.bin"
+	// pythonPath = os.Getenv("HOME") + "/.local/share/gosplat/src/python_helper/fast_model_compare.py"
```

Comment:
Should be updated to the paths it was before after done

Reply:
Will revert after PR is done and local testing is completed. :+1: